cve_bin_tool/OutputEngine.py

```diff
@@ -137,19 +150,19 @@ def output_console(self, outfile):
             +=================================================================+
 
             +=================================================================+
-            |   MODULE NAME      |  VERSION  |    CVE NUMBER      | SEVERITY  |
+            |   PRODUCT NAME     |  VERSION  |    CVE NUMBER      | SEVERITY  |
```

Comment:
Doesn't need to be updated in this patch, but we should probably plan to show vendor where we show product, since the two together are more or less what we're using as a unique key into NVD.